react/tic-tac-toe/src/app/screens/Game/components/Square/index.js

```diff
@@ -1,11 +1,19 @@
+/* eslint-disable react/jsx-no-bind */
+/* eslint-disable react/prop-types */
```

Comment:
Remove both diabled. You can start the next card to fix prop-types

Reply:
But... on this way `husky` throw an error :c